tests/test-text.lua

```diff
@@ -46,18 +46,19 @@ batteries. That is because
 the goal of Lua is to
 produce a lean expressive
 language that will be
-used on all sorts of machines,
```

Comment:
length of `used on all sorts of` = 20
should have jumped instead of putting `machines,` on same line